src/transformState.js

```diff
@@ -61,8 +61,33 @@
  * @param {Object} state
  * @param {Object[]} transforms
  */
-function transformState(state, transforms) {
-  // write code here
+
+function transformState(state, transform) {
+  for (const transforms of transform) {
+    switch (transforms.operation) {
+      case 'addProperties' : Object.assign(state, transforms.properties);
+        break;
+
+      case 'removeProperties':
+        for (const key of transforms.properties) {
+          delete state[key];
+        };
+        break;
+
+      case 'clear':
+        for (const key in state) {
+          delete state[key];
+        }
+
+        break;
```

Comment:
```suggestion
        }
        break;
```
Stick to one style of formatting in your code. In other cases there are no empty lines before 'break', but here you insert it for some reason